docs/src/inscriptions.md

```diff
@@ -62,3 +62,40 @@ The inscription content is contained within the input of a reveal transaction,
 and the inscription is made on the first sat of its input. This sat can
 then be tracked using the familiar rules of ordinal theory, allowing it to be
 transferred, bought, sold, lost to fees, and recovered.
+
+Content
+-------
+
+The data model of inscriptions is that of a HTTP response, allowing inscription
+content to be server by a web server and viewed in a web browser.
```

Comment:
A small typo here. Should be "content to be **served** by a web server".

Reply:
Thank you! Fixed.